### ISSUE TITLE
Task List

### ISSUE THREAD

**Issue by theogruner** [theogruner/Project_RL#1]
- [x] try batch normalization on DDPG neural nets
